articles/service-fabric/service-fabric-backup-restore-service-trigger-restore.md

```diff
@@ -60,20 +60,28 @@ For the following example, assume that the lost cluster is the same cluster that
 
 #### Powershell using Microsoft.ServiceFabric.Powershell.Http Module
 
+Replace `account-name` with your storage account name.
+
 ```powershell
-Get-SFBackupsFromBackupLocation -Application -ApplicationName 'fabric:/SampleApp' -AzureBlobStore -ConnectionString 'DefaultEndpointsProtocol=https;AccountName=<account-name>;AccountKey=<account-key>;EndpointSuffix=core.windows.net' -ContainerName 'backup-container'
+
+    Get-SFBackupsFromBackupLocation -Application -ApplicationName 'fabric:/SampleApp' -ManagedIdentityAzureBlobStore -BlobServiceUri "https://<account-name>.blob.core.windows.net" -ContainerName 'backup-container' -ManagedIdentityType "VMSS" -FriendlyName "AzureMI_storagesample" -ManagedIdentityClientId = "<clien-ID of User-Assigned MI>"
+
+    # Use Optional parameter `ManagedIdentityClientId` with clien-ID of User-Assigned Managed Identity in case of multiple User-Assigned managed identities assigned to your resource, else no need of this paramter.
 
 ```
 
 #### Rest Call using Powershell
 
-Execute a PowerShell script to use the REST API to return a list of the backups created for all partitions inside the `SampleApp` application. The API requires the backup storage information to list the available backups.
+Execute a PowerShell script to use the REST API to return a list of the backups created for all partitions inside the `SampleApp` application. The API requires the backup storage information to list the available backups. Replace `account-name` with your storage account name.
 
 ```powershell
 $StorageInfo = @{
-    ConnectionString = 'DefaultEndpointsProtocol=https;AccountName=<account-name>;AccountKey=<account-key>;EndpointSuffix=core.windows.net'
-    ContainerName = 'backup-container'
-    StorageKind = 'AzureBlobStore'
+    StorageKind = "ManagedIdentityAzureBlobStore"
+    FriendlyName = "AzureMI_storagesample"
+    BlobServiceUri = "https://<account-name>.blob.core.windows.net"
+    ContainerName = "backup-container"
+    ManagedIdentityType = "VMSS"
+    ManagedIdentityClientId = "<clien-ID of User-Assigned MI>"  # Use Optional parameter `ManagedIdentityClientId` with clien-ID of User-Assigned Managed Identity in case of multiple User-Assigned managed identities assigned to your resource, else no need of this paramter.
```

Comment:
Same needs to be clear when to use ManagedIdentityClientId

Reply:
updated